ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/usertags/customer-managed-key-sc/provision/cucushift-installer-rehearse-azure-ipi-usertags-customer-managed-key-sc-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/usertags/customer-managed-key-sc/provision/cucushift-installer-rehearse-azure-ipi-usertags-customer-managed-key-sc-provision-chain.yaml",
```

Comment:
What does `sc` stand for?

Reply:
ah, should be sa(storage account), will update when job is completed.

Reply:
I changed `customer-managed-key-sc` to `storage-account-encryption`, this might be more meaningful.

Reply:
Can the customer manager key be used to encrypt masters, workers instances?

Reply:
For master, workers disk encryption, it requires disk encryption set besides customer managed key.